server/database/config/db_build.sql

```diff
@@ -95,8 +95,8 @@ CREATE TABLE member_skill (
 
 CREATE TABLE filter (
   member_id INTEGER REFERENCES member(id),
-  skills VARCHAR,
-  offer_type VARCHAR
+  skills VARCHAR[],
+  offer_type VARCHAR[]
```

Comment:
@Angham116 There's also a mysql JSON data type (https://dev.mysql.com/doc/refman/8.0/en/json.html) which could work for this.  No need to change now, just something you could try out for the next app :)